modules/mobile/docs/openapi.yaml

```diff
@@ -3713,6 +3762,12 @@ components:
         default: snake
       description: Allows the API to return camelCase keys rather than snake_case
   responses:
+    '400':
```

Comment:
I added these two to the new approach but didn't with some of the less common error types because I don't know if they'll be used elsewhere. We can always change things later.